tests/Aspire.Hosting.Kafka.Tests/KafkaFunctionalTests.cs

```diff
@@ -101,20 +97,34 @@ public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
             else
             {
                 bindMountPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+
+                Directory.CreateDirectory(bindMountPath);
+
+                if (!OperatingSystem.IsWindows())
+                {
+                    // the docker container runs as a non-root user, so we need to grant other user's read/write permission
+                    // to the bind mount directory.
+                    // Note that we need to do this after creating the directory, because the umask is applied at the time of creation.
+                    const UnixFileMode BindMountPermissions =
+                        UnixFileMode.UserRead | UnixFileMode.UserWrite | UnixFileMode.UserExecute |
+                        UnixFileMode.GroupRead | UnixFileMode.GroupWrite | UnixFileMode.GroupExecute |
+                        UnixFileMode.OtherRead | UnixFileMode.OtherWrite | UnixFileMode.OtherExecute;
+
+                    File.SetUnixFileMode(bindMountPath, BindMountPermissions);
```

Comment:
Is this something we need to do in the kafka extension and not in the tests? Pgadmin does something similar:

https://github.com/dotnet/aspire/blob/dad2369a4584dcf1b92d1ba957d424b18c373dbd/src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs#L194-L198

Reply:
I guess this is auto generated.